AMANDA/amanda_to_s3.py

```diff
@@ -60,7 +60,7 @@
         OR(foldertype in('RW', 'EX')
             AND STATUSCODE NOT IN(70045, 50003)
             AND INDATE >= TO_DATE('10-01-2018', 'mm-dd-yyyy')
-            AND SUBCODE NOT IN(50510)
+            AND SUBCODE NOT IN(50510, 50505)
```

Comment:
ignoring film permits as well now as ROW no longer manages these.

Reply:
just being nosy, i wonder who handles the film permits now...

Reply:
office of special events I believe @chiaberry